site/assets/scss/_navbar.scss

```diff
@@ -13,20 +13,10 @@
 
   // Boosted mod: no .navbar-brand
 
-  // Boosted mod: Needed since the dropdowns define their own color (could be removed once they are tackled)
-  .nav-link {
-    &:hover {
-      color: var(--bs-link-hover-color);
-    }
-
-    &.active {
-      color: var(--bs-link-hover-color);
-    }
-
-    .bi {
-      width: var(--bd-navbar-dropdown-item-icon-size);
-      height: var(--bd-navbar-dropdown-item-icon-size);
-    }
+  // Boosted mod: Needed since our icons are larger in the navbar
```

Comment:
Not sure about this whole change for the icons. We checked with the designers the rendering of the icons within the dropdown which is changed here. Now we have really big icons:

![Screenshot 2023-11-15 at 08 00 48](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/assets/17381666/8b0890b0-0a22-4d2d-b627-ac6873954d95)

Maybe let's wait for the design specifications of this theme selector before changing the actual rendering.

Reply:
I think that they are the same size as before, it should be transparent removal